### PR TITLE
AK/2.6 -> CCS/2.6 Manual Merge

### PR DESCRIPTION
**Pull request by andrewegel**
The Sync Job was stuck on `gradle/dependencies.gradle` around the jetty version. I'm specifically paging @ijuma to review this since he added the comment block around it:

```
-   // 9.4.25 renamed closeOutput to completeOutput (https://github.com/eclipse/jetty.project/commit/c5acf965067478784b54e2d241ec58fdb0b2c9fe)
-   // which is a method used by recent Jersey versions when this comment was written (2.30.1 was the latest). Please
-   // verify that this is fixed in some way before bumping the Jetty version.
```

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by andrewegel** [comment]
This build failed on downstream common verification (due to what looks like a network blip): https://jenkins.confluent.io/job/Confluentinc%20Contributors/job/kafka/job/PR-458/6/ but all ce-kafka tests passed. This job https://jenkins.confluent.io/job/Confluentinc%20Contributors/job/kafka/job/PR-458/4/ all downstream versifiers passed, but failed on a single test that passed in run 6. 

IMO this indicates the merge is good, just need sign off from someone from Kafka.

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -68,11 +68,8 @@ versions += [
   easymock: "4.2",
   jackson: "2.10.5",
   jacoco: "0.8.5",
-  // 9.4.25 renamed closeOutput to completeOutput (https://github.com/eclipse/jetty.project/commit/c5acf965067478784b54e2d241ec58fdb0b2c9fe)
-  // which is a method used by recent Jersey versions when this comment was written (2.30.1 was the latest). Please
-  // verify that this is fixed in some way before bumping the Jetty version.
-  jetty: "9.4.24.v20191120",
-  jersey: "2.30",
+  jetty: "9.4.33.v20201020",
+  jersey: "2.31",
```
Comment (stanislavkozlovski) [inline gradle/dependencies.gradle:72]: Seems like Jersey 2.31 has a fix for this - https://github.com/eclipse-ee4j/jersey/commit/950ce6ce91bd7aafeabb939c487f72f73e77dc71

Additionally, AK removed this comment with commit https://github.com/apache/kafka/commit/492306a0c5ad558d8acf34325985aed803ae75fd in June
For the 2.6 branch, it was removed recently in Nov (https://github.com/apache/kafka/commit/df17166072d61be410da48c46db0a87e7e99a51f) (https://github.com/apache/kafka/pull/9556)

**Review by stanislavkozlovski — APPROVED** [review]
LGTM!

File: gradle/dependencies.gradle (hunk 1 above)
Comment (ijuma) [reply gradle/dependencies.gradle:72]: Yeah, that's right.

**Pull request merged**